examples/src/examples/graphics/normals-and-tangents.mjs

```diff
@@ -0,0 +1,121 @@
+    };
+
+    // Set the canvas to fill the window and automatically change resolution to be the same as the canvas size
+    app.setCanvasFillMode(pc.FILLMODE_FILL_WINDOW);
+    app.setCanvasResolution(pc.RESOLUTION_AUTO);
+
+    // Ensure canvas is resized when window changes size
+    const resize = () => app.resizeCanvas();
+    window.addEventListener('resize', resize);
+    app.on('destroy', () => {
+        window.removeEventListener('resize', resize);
+    });
+
+    /**
+     * @param {pc.Asset[] | number[]} assetList - The asset list.
+     * @param {pc.AssetRegistry} assetRegistry - The asset registry.
+     * @returns {Promise<void>} The promise.
+     */
+    function loadAssets(assetList, assetRegistry) {
+        return new Promise(resolve => {
+            const assetListLoader = new pc.AssetListLoader(assetList, assetRegistry);
+            assetListLoader.load(resolve);
+        });
+    }
+    await loadAssets(Object.values(assets), app.assets);
+
+    // Setup skydome
+    app.scene.envAtlas = assets.helipad.resource;
+    app.scene.toneMapping = pc.TONEMAP_ACES;
+    app.scene.skyboxMip = 2;
+    app.scene.skyboxRotation = new pc.Quat().setFromEulerAngles(0, 70, 0);
+    app.scene.skyboxIntensity = 1.5;
+
+    // Create an entity hierarchy representing the statue
+    const statueEntity = assets.statue.resource.instantiateRenderEntity();
+    statueEntity.setLocalEulerAngles(0, 90, 0);
+    app.root.addChild(statueEntity);
+
+    // Create a camera with an orbit camera script
+    const camera = new pc.Entity();
+    camera.addComponent("camera");
+    camera.addComponent("script");
+    camera.script.create("orbitCamera", {
+        attributes: {
+            inertiaFactor: 0 // Override default of 0 (no inertia)
+        }
+    });
+    camera.script.create("orbitCameraInputMouse");
+    camera.script.create("orbitCameraInputTouch");
+    app.root.addChild(camera);
+    camera.script.orbitCamera.pitch = 0;
+    camera.script.orbitCamera.yaw = 90;
+    camera.script.orbitCamera.distance = 4;
+
+    const directionalLight = new pc.Entity();
+    directionalLight.addComponent("light", {
+        type: "directional",
+        color: pc.Color.WHITE,
+        castShadows: true,
+        intensity: 1,
+        shadowBias: 0.2,
+        normalOffsetBias: 0.05,
+        shadowResolution: 2048
+    });
+    directionalLight.setEulerAngles(45, 180, 0);
+    app.root.addChild(directionalLight);
+
+    app.start();
+    return app;
+}
+
+class NormalsAndTangentsExample {
+    static CATEGORY = 'Graphics';
+    static WEBGPU_ENABLED = true;
+    static DESCRIPTION = '<a href="https://github.com/KhronosGroup/glTF-Sample-Models/tree/main/2.0/NormalTangentTest">Model and textures by Ed Mackey</a>, copyright 2017-2018 Analytical Graphics Inc. CC-BY 4.0 <a href="https://creativecommons.org/licenses/by/4.0/">https://creativecommons.org/licenses/by/4.0/</a>.';
```

Comment:
Depends on https://github.com/playcanvas/engine/pull/5922 to show the copyright in the description.

Reply:
We don't need to display the attribution, we just need to have it stored in the file along with the glb. See this as an example of what we do: https://github.com/playcanvas/engine/blob/main/examples/assets/models/IridescentDishWithOlives.txt